app/views/service_providers/_form.html.erb

```diff
@@ -281,7 +281,7 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
                  label_html: { class: 'usa-input-required'},
                  label: 'Failure to Proof URL',
                  hint: I18n.t('service_provider_form.failure_to_proof_url'),
-                 required: form.object.ial == 2 ? true : false %>
+                 required: true %>
```

Comment:
note: the actual `required` value is determined via the javascript toggling on page load. marking it `true` in the erb ensures that the label `*required` is added to the input correctly. (see screenshot)

<img width="511" alt="failure_to_proof_url input with required label" src="https://github.com/18F/identity-dashboard/assets/5473830/d5b5577b-3043-4034-8766-4c0e56045e66">